style.css

```diff
@@ -0,0 +1,639 @@
+  font-size: 13px;
+  font-weight: lighter;
+}
+
+.subscribe-link p {
+  position: relative;
+  left: 250px;
+  margin: 5px 0;
+}
+
+.subscribe-link-section {
+  color: #000;
+}
+
+.mid-links div {
+  flex: 1;
+  text-decoration: none;
+  color: #000;
+}
+
+.mid-links .li-bold {
+  font-weight: bolder;
+  font-size: 16px;
+  text-decoration: none;
+  color: #000;
+}
+
+.links .mid-links div:last-child {
+  border-left: 1px solid #e2e2e2;
+  padding-left: 15px;
+}
+
+.links .mid-links div:last-child ul li:nth-child(7) {
+  border-bottom: 1px solid #e2e2e2;
+}
+
+.bottom-social-media-icons {
+  margin-top: 30px;
+  margin-bottom: 100px;
+  justify-content: space-between;
+}
+
+.bottom-social-media-icons button {
+  padding: 7px 12px;
+  background-color: #466880;
+  border-radius: 5px;
+}
+
+/* spacs-astronomy */
+.grid {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.grid-1 {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.grid-section-title {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 70px;
+  position: relative;
+  bottom: 10px;
+}
+
+.grid-section-title-hr-astronomy {
+  width: 850px;
+  position: relative;
+  bottom: 30px;
+  height: 3px;
+  background-color: #000;
+}
+
+.astronomy-title {
+  position: relative;
+  left: 10px;
+}
+
+.gird-section-most-popular-hr {
+  width: 340px;
+  position: relative;
+  bottom: 30px;
+  height: 3px;
+  background-color: #000;
+}
+
+.grid-section-title h3 {
+  font-size: 16px;
+}
+
+.grid-section {
+  display: grid;
+  grid-auto-rows: 600px;
+  grid-template-columns: 70% 30%;
+  grid-gap: 1.5em;
+  margin-left: 10px;
+}
+
+.space-astronomy {
+  display: grid;
+  background-color: white;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 10px;
+}
+
+.most-popular {
+  display: grid;
+  background-color: white;
+  grid-template-rows: repeat(10, 1fr);
+}
+
+.most-popular-hr {
+  position: relative;
+  right: 7px;
+  top: 10px;
+}
+
+.editor-pick-section {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.editor-pick-section .editors-pick-images-section img {
+  width: 280px;
+}
+
+.editors-pick-images-section {
+  margin-right: 10px;
+}
+
+.editors-picks-title-hr {
+  margin-top: 20px;
+}
+
+.Editors-picks-hr {
+  width: 850px;
+  position: relative;
+  bottom: 30px;
+  font-weight: bolder;
+  background-color: #000;
+  height: 3px;
+}
+
+.editors-picks-grid-title {
+  font-size: 16px;
+  margin: 0 0 10px 10px;
+}
+
+.goto {
+  color: #000;
+  font-size: 12px;
+}
+
+/* footer */
+
+.links .links-end ul {
+  justify-content: space-between;
+  padding: 10px 80px;
+}
+
+.links .links-end ul a {
+  text-decoration: none;
+  color: #666;
+  font-size: 11px;
+}
+
+.links .links-end ul a:not(:last-child) {
+  margin-left: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+footer {
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border-top: 1px solid black;
+  background-color: #fff;
+}
+
+.footer-container {
+  width: 1320px;
+  margin: 0 auto;
+  padding: 20px 0;
+}
```

Comment:
- [ ] **MANDATORY** For good CSS practices, delete inefficient selectors. You are not using this class in your HTML, please remove it